packages/rolldown/src/options/output-options.ts

```diff
@@ -23,6 +23,19 @@ export type ChunkFileNamesFunction = (chunkInfo: PreRenderedChunk) => string
 
 export type GlobalsFunction = (name: string) => string
 
+export type ESTarget =
+  | 'ES2015'
```

Comment:
```suggestion
  | 'es2015'
```

and should add an extra `es6` as a alias for `es2015`.